From 8b4db22b3770695c58eae44e307b8d615b1e70af Mon Sep 17 00:00:00 2001
From: grotaclas <grotaclas@gmail.com>
Date: Tue, 21 Oct 2025 17:24:50 -0000
Subject: [PATCH] release VOC with idea groups filled and correct national
 ideas

because they were released with cede_province, they didn't have any idea
groups and because they had an unconditional swap_free_idea_group, their
national ideas changed to colonial_ideas(because they normally already
have voc_ideas). Now they are created with release = VOC, which also
seems to give them the province discoveries of their parent country(If I
see it correctly they only saw the provinces which they could see due to
their tech group/religion).

Bugreport: https://forum.paradoxplaza.com/forum/threads/1-37-voc-spawns-with-0-ideas-filled.1675302/
---
 decisions/VOC_decisions.txt | 11 ++++++++---
 1 file changed, 8 insertions(+), 3 deletions(-)

diff --git a/decisions/VOC_decisions.txt b/decisions/VOC_decisions.txt
index a3108db8..c26410bc 100755
--- a/decisions/VOC_decisions.txt
+++ b/decisions/VOC_decisions.txt
@@ -21,10 +21,9 @@ country_decisions = {
 					limit = {
 						 superregion = india_superregion
 					}
-					cede_province = VOC
 					add_core = VOC
 				}
-				
+				release = VOC
 				create_subject = {
 					subject_type = commercial_enterprise
 					subject = VOC
@@ -40,7 +39,13 @@ country_decisions = {
 					add_treasury = 200
 					set_prestige_to_zero = yes
 					add_republican_tradition = 100
-					swap_free_idea_group = yes
+					if = {
+						limit = {
+							NOT = { has_idea_group = voc_ideas }
+						}
+						swap_free_idea_group = yes
+					}
+
 					adopt_technology_group_from_target = { target = root }
 				}
 				india_superregion = { 
-- 
2.48.1

